Sources/SmileID/Classes/BiometricKYC/OrchestratedBiometricKycViewModel.swift

```diff
@@ -142,7 +142,16 @@ class OrchestratedBiometricKycViewModel: ObservableObject {
         if let livenessFiles {
             allFiles.append(contentsOf: livenessFiles)
         }
-        return try LocalStorage.zipFiles(at: allFiles)
+        do {
+            let securityInfo = try createSecurityInfo(files: allFiles)
+            return try LocalStorage.zipFiles(
+                urls: allFiles,
+                data: ["security_info.json": securityInfo]
+            )
+        } catch {
```

Comment:
Same here.

Reply:
Resolved in [f1c0a73](https://github.com/smileidentity/ios/pull/286/commits/f1c0a7342480f07515b317fc20349c7a7ddf0bd5)